README.md

```diff
@@ -174,6 +177,48 @@ This repository follows the snakemake guidelines for reproducibility:
     ├── results
     └── resources
 
+## Recreate Paper Results
+
+To recreate the results shown in the accompanying [sensitivity analysis paper](https://doi.org/10.12688/openreseurope.15461.1), follow the steps below. 
+
+1. Pull all data associated with the paper using snakemake 
+```
+snakemake setup_paper_data -c --config skip_checks
```

Comment:
This command does not work.  `-c` is related to a cluster.  Should it be `--configfile skip_checks` (but then the `skip_checks` file does not exist)?

Reply:
I always use `-c` for executing workflows; never knew it was specific for clusters. I guess we should change that to `-j` or `-j6` or something?

Reply:
Changed command to `j6`